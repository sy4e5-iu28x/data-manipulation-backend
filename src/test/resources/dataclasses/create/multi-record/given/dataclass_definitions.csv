id,name
1,クラス1
2,クラス2