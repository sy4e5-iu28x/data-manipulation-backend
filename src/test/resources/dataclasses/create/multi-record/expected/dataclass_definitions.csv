id,name,type
1,クラス1,String
2,クラス2,Number
3,新しいクラス,UserDefinedClass